content/cif-connector/tests/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "cif-connector",
+  "private": true,
+  "version": "0.0.1",
+  "description": "CIF Connector - Content Package",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run prettier:check && npm run karma:run",
+    "karma:run": "karma start --single-run --browsers ChromeHeadless,FirefoxHeadless karma.conf.js",
+    "karma:debug": "karma start --single-run=false --browsers Chrome --debug karma.conf.js",
+    "prettier:check": "prettier --check '{../src,src}/**/*.{js,css}' --config ./.prettierrc",
+    "prettier:fix": "prettier --write '{../src,src}/**/*.{js,css}' --config ./.prettierrc",
+    "webpack:build": "webpack --config webpack.config.js --env.production --mode development"
+  },
+  "author": "Adobe",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/aem-core-cif-components.git"
+  },
+  "browserslist": [
```

Comment:
This property is only used for babel transpiling which you also don't use anywhere, so this should be removed as well.